README.md

```diff
@@ -33,6 +33,14 @@ See the [usage documentation](docs/en/index.md) for more information.
     composer: `composer require silverstripe/secureassets dev-master`
  2. Run /dev/build?flush=1
 
+## Upgrade Instructions
+As above, this module is no longer needed in the SilverStripe 4.x release line. When you upgrade your project from the 
+SilverStripe 3.x to SilverStripe 4.x, ensure you export the database from your SS3 project and import it 
+into your SS4 project. 
+
+Run `dev/tasks/MigrationTask` followed by `dev/build`. All secure assets will be migrated to 
```

Comment:
I think you should suggest that devs run `dev/build` first, then `dev/tasks/MigrateFileTask` (note task name) afterwards

Reply:
Thanks, I made the requested updates.